src/database/remove-remote.ts

```diff
@@ -0,0 +1,128 @@
+import { Response } from "request";
```

Comment:
We aren't 100% consistent today, but please rename this to `removeRemote.ts` as that's closest to existing file naming patterns in the CLI.